src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala

```diff
@@ -83,4 +84,22 @@ class SapMachineMigrations {
       platform,
       Some(SAP)
     )
+
+  @ChangeSet(
+    order = "0012",
+    id = "0012-add_sapmchn_jdk_15",
+    author = "poad"
+  )
+  def migrate0012(implicit db: MongoDatabase) =
+    Map(
+      Linux64 -> "linux-x64_bin.tar.gz",
+      MacOSX  -> "osx-x64_bin.tar.gz",
+      Windows -> "windows-x64_bin.zip"
+    ).map {
+        case (platform, suffix) =>
+          toVersion(platform, suffix, "15")
```

Comment:
```suggestion
          toVersion(platform, suffix, "15.0.0")
```

Also think we should not mix the identifier with the version used by the vendor

Reply:
Fixed it